src/handlers/comment/handlers/assign.ts

```diff
@@ -63,6 +68,7 @@ export const assign = async (body: string) => {
   const duration = targetTimeLabel.value;
   if (!duration) {
     logger.info(`Missing configure for timelabel: ${targetTimeLabel.name}`);
+    await addCommentToIssue("Skipping /assign since configuration is missing for the following labels", issue_number);
```

Comment:
```suggestion
    await addCommentToIssue("Skipping `/assign` since configuration is missing for the following labels", issue_number);
```

You should monospace (highlight) all bot commands in the comments.